### PR TITLE
Added code-based experiences docs

### PR DESCRIPTION
**Pull request by swarna04**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/edge/adobe-journey-optimizer/public-classes/tabs/surface.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+noIndex: true
+---
+
+<Variant platform="ios" api="init" repeat="6"/>
+
+#### Swift
+
+#### Syntax
```
Comment (calebk1) [inline src/pages/edge/adobe-journey-optimizer/public-classes/tabs/surface.md:9]: Same comment again for H4s.

**Review by calebk1 — CHANGES_REQUESTED** [review]
Content looks good, but a few header changes needs to be done.

File: src/pages/edge/adobe-journey-optimizer/public-classes/proposition-item.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+# PropositionItem
+
+The `PropositionItem` class represents the decision proposition item received from the remote, upon a personalization query to the Experience Edge network.
+
+## iOS Interface - PropositionItem
+
+## Public variables
+
+### itemId
```
[the remark this answers is not in the document]
Comment (spoorthipujariadobe) [reply src/pages/edge/adobe-journey-optimizer/public-classes/proposition-item.md:23]: Moving it down an H level would hide it from the right navigation highlights which appears under "**ON THIS PAGE**"

[the remark this answers is not in the document]
Comment (spoorthipujariadobe) [reply src/pages/edge/adobe-journey-optimizer/public-classes/proposition-item.md:23]: I can add a line of content in between the two H2 if that helps?

**Comment by sbenedicadb** [comment]
closing this pr and pulling in the changes in #539

**Pull request closed without merging**